common/src/test/java/com/netflix/conductor/common/workflow/WorkflowDefTest.java

```diff
@@ -128,7 +128,7 @@ public void testWorkflowDefConstraints() {
 	@Test
 	public void testWorkflowDefConstraintsWithMultipleEnvVariable() {
 		WorkflowDef workflowDef = new WorkflowDef();//name is null
```

Comment:
Not related to this PR, but this test could reuse WorkflowDef creation logic using @before or other constructs.